specs/deneb/p2p-interface.md

```diff
@@ -146,18 +156,19 @@ New validation:
 
 This topic is used to propagate signed blob sidecars, where each blob index maps to some `subnet_id`.
 
-The following validations MUST pass before forwarding the `signed_blob_sidecar` on the network, assuming the alias `sidecar = signed_blob_sidecar.message`:
+The following validations MUST pass before forwarding the `blob_sidecar` on the network, assuming the alias `sidecar = blob_sidecar` and `block_header = blob_sidecar.signed_block_header.message`:
```

Comment:
```suggestion
The following validations MUST pass before forwarding the `sidecard` on the network, assuming the alias `block_header = sidecar.signed_block_header.message`:
```

There is no need to alias `blob_sidecar` to `sidecar`. I think it's better to have only one variable to avoid confusion.